go.mod

```diff
@@ -169,7 +169,7 @@ require (
 
 replace (
 	github.com/nxadm/tail => github.com/nxadm/tail v0.0.0-20211216163028-4472660a31a6
-	github.com/openshift/api => github.com/openshift/api v0.0.0-20191219222812-2987a591a72c
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
```

Comment:
Should be this change part of this PR?

Reply:
It has to be, workspaces do not allow different versions when replacing dependencies.